src/services/findAddress.js

```diff
@@ -0,0 +1,8 @@
+import createNewHouse from '../models/createNewHouseModel'
+
+export const findAddress = async (addressData) => {
```

Comment:
It doesn't work exactly as expected. The error will be thrown if someone tries to save a new house with an address already in the database but if a not required part of the address is skipped next time, the house will be saved. Checking if a house hasn't been added would probably have to rely on a subset of address data consisting only required fields.